usr/src/lib/libzfs/common/libzfs_pool.c

```diff
@@ -1884,6 +1884,26 @@ zpool_scan(zpool_handle_t *zhp, pool_scan_func_t func)
 }
 
 /*
+ * Trim the pool.
+ */
+int
+zpool_trim(zpool_handle_t *zhp, boolean_t start, uint64_t rate)
+{
+	zfs_cmd_t zc = { 0 };
+	char msg[1024];
+	libzfs_handle_t *hdl = zhp->zpool_hdl;
+	trim_cmd_info_t tci = { .tci_start = start, .tci_rate = rate };
+
+	(void) strlcpy(zc.zc_name, zhp->zpool_name, sizeof (zc.zc_name));
+	zc.zc_cookie = (uintptr_t)&tci;
+
+	if (zfs_ioctl(hdl, ZFS_IOC_POOL_TRIM, &zc) == 0)
```

Comment:
I'd prefer to keep the file consistent and recommend we do a full replacement pass across the entire file in a later commit.